recipes/mpir/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+chmod +x configure
+
+./configure --prefix=$PREFIX --enable-cxx
+
+make
+
+if [ "$(uname)" != "Darwin" ];
+then
+    make check
```

Comment:
Because in staged-recipes we build every possible combination of Python and NumPy that is supported by the recipe. However, we can't know which ones will be needed in advance so we use one build to do the whole job. Once this is converted to a feedstock that won't be necessary any more as each version will be a separate item in a build matrix (with the exception of Circle CI). So, keep the tests commented for now and we can re-enable them when this goes into a feedstock.